examples/kubernetes/deployment+service.privileged.yaml

```diff
@@ -14,6 +14,15 @@ spec:
       labels:
         app: buildkitd
     spec:
+      initContainers:
+      - name: qemu
+        image: 'tonistiigi/binfmt:latest'
+        args:
+          - '--install'
+          - all
+        imagePullPolicy: Always
+        securityContext:
+          privileged: true
```

Comment:
Usually you shouldn't need binfmt when you are using the upstream `moby/buildkit` image

Reply:
@AkihiroSuda Looks like this stalled. You still want changes or prefer this to be closed?